amd_openvx_extensions/CMakeLists.txt

```diff
@@ -24,23 +24,39 @@ project(amd_openvx_extensions)
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../amd_openvx/cmake)
 
 if(GPU_SUPPORT)
+    find_package(AMDRPP QUIET)
     if("${BACKEND}" STREQUAL "OPENCL")
         find_package(OpenCL QUIET)
         if(OpenCL_FOUND)
             SET(BUILD_OPENCL True)
             message("-- ${Blue}amd_openvx_extensions BACKEND set to ${BACKEND}${ColourReset}")
             find_package(OpenCV QUIET)
             find_package(FFmpeg QUIET)
-            find_package(AMDRPP QUIET)
             find_package(miopengemm PATHS ${ROCM_PATH} QUIET)
             find_package(miopen     PATHS ${ROCM_PATH} QUIET)
         else()
             set(GPU_SUPPORT OFF)
             set(BUILD_OPENCL False)
         endif()
+    elseif("${BACKEND}" STREQUAL "HIP")
+        if(NOT DEFINED HIP_PATH)
+            if(NOT DEFINED ENV{HIP_PATH})
+                set(HIP_PATH ${ROCM_PATH}/hip CACHE PATH "Path to which HIP has been installed")
+            else()
+                set(HIP_PATH $ENV{HIP_PATH} CACHE PATH "Path to which HIP has been installed")
+            endif()
+        endif()
+        list(APPEND CMAKE_MODULE_PATH ${HIP_PATH}/cmake)
+        # HSA_PATH
+        IF(NOT DEFINED ENV{HSA_PATH})
+            SET(HSA_PATH ${ROCM_PATH}/hsa)
+        ELSE()
+            SET(HSA_PATH $ENV{HSA_PATH})
+        ENDIF()
+        find_package(HIP QUIET REQUIRED)
     else()
         SET(BUILD_OPENCL False)
```

Comment:
add set(GPU_SUPPORT OFF) here as well.

Reply:
HCC_PATH is not required to set anymore with newer compiler

Reply:
Done.